pkg/apis/istio/v1alpha2/validation.go

```diff
@@ -58,20 +54,20 @@ func validateSubTypes(e reflect.Value, failOnMissingValidation bool, values *Val
 		}
 		val := e.Field(i).Elem()
 		if val == reflect.ValueOf(nil) {
-			fmt.Printf("element: %s is not defined\n", e.Type().Field(i).Name)
 			continue
 		}
 		validation := e.Field(i).MethodByName("Validation")
 		if validation == reflect.ValueOf(nil) {
```

Comment:
You can use util.IsNilOrInvalidValue here

Reply:
done